client/src/components/hooks/useSquidDetails.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery, UseQueryResult } from "react-query";
+import { AxiosError } from "axios";
+
+import { ApiClient } from "../../backend/ApiClient";
+
+// DRY and re-use for multiple interfaces
```

Comment:
When you have an interface that refers to an object like this that will be used elsewhere, consider moving it to e.g., `/src/models/Squid.d.ts`. 

This is a pattern that we're still very much figuring out, but I understand a `d.ts` file to basically be a file exclusively for typescript definitions. If you had also implemented a Squid POJO, it would be more appropriate to put the interface in the same file as the model it describes (`/src/models/Squid.ts`)